axelrod/tests/strategies/test_player.py

```diff
@@ -359,9 +359,6 @@ def test_responses(test_class, player1, player2, responses, history1=None,
     for response in responses:
         s1, s2 = simulate_play(player1, player2)
         test_class.assertEqual(s1, response)
-    if attrs:
```

Comment:
(The coverage on this PR dropped because that line wasn't being hit by anything, which is why I just removed it.)